docs/4-user-guide/2-zarf-packages/3-the-zarf-init-package.md

```diff
@@ -4,36 +4,41 @@ sidebar_position: 3
 
 # The Zarf 'init' Package
 
-The init package is the `zarf.yaml` file that lives at the [root of the Zarf repository](https://github.com/defenseunicorns/zarf/blob/main/zarf.yaml). It is defined via composed components that all offer value for future packages to utilize. When the init package is deployed, it will create a `zarf` namespace within your k8s cluster and deploy various pods, services, and secrets to that namespace, depending on which optional components you choose to deploy.
+The init package is the `zarf.yaml` file that lives at the [root of the Zarf repository](https://github.com/defenseunicorns/zarf/blob/main/zarf.yaml).
+It is defined by composed components which provide a foundation for future packages to utilize. Upon deployment, the init package generates a `zarf` namespace within your K8s cluster and deploys pods, services, and secrets to that namespace based on the optional components selected for deployment.
 
 ## Mandatory Components
 
-Zarf's work necessitates that some components in the [init package](https://github.com/defenseunicorns/zarf/blob/main/zarf.yaml) are "always on" (a.k.a. required & cannot be disabled). These components are always deployed whenever you perform a `zarf init` command. Those include:
+Zarf's capabilities require certain components of the init package to be constantly active, meaning that they cannot be disabled and are always on. These components are automatically deployed whenever a `zarf init` command is executed. These components include:
```

Comment:
I think this part of the docs is just really old as this is not accurate anymore.  The only part of the Zarf init package that is required (and even that is loose because there are scenarios that do not use it) is the `zarf-agent`.  The Zarf agent is the mutating webhook that takes pod specs and flux GitRepository objects and modifies them to match their air gap equivalents.

In _most_ scenarios Zarf will also deploy an internal registry (using the components described below), however Zarf can be configured to use an already existing registry with the `--registry-*` flags on `zarf init`).  Doing that will skip the injector and see process and will not deploy a registry to the cluster instead uploading any images to the externally configured registry.